src/univariate/continuous/betaprime.jl

```diff
@@ -32,15 +32,18 @@ struct BetaPrime{T<:Real} <: ContinuousUnivariateDistribution
     BetaPrime{T}(α::T, β::T) where {T} = new{T}(α, β)
 end
 
-function BetaPrime(α::T, β::T; check_args=true) where {T<:Real}
+function BetaPrime(α::T, β::T; check_args::Bool=true) where {T<:Real}
     check_args && @check_args(BetaPrime, α > zero(α) && β > zero(β))
     return BetaPrime{T}(α, β)
 end
 
-BetaPrime(α::Real, β::Real) = BetaPrime(promote(α, β)...)
-BetaPrime(α::Integer, β::Integer) = BetaPrime(float(α), float(β))
-BetaPrime(α::Real) = BetaPrime(α, α)
-BetaPrime() = BetaPrime(1.0, 1.0, check_args=false)
+BetaPrime(α::Real, β::Real; check_args::Bool=true) = BetaPrime(promote(α, β)...; check_args=check_args)
+BetaPrime(α::Integer, β::Integer; check_args::Bool=true) = BetaPrime(float(α), float(β); check_args=check_args)
+function BetaPrime(α::Real; check_args::Bool=true)
+    check_args && @check_args(BetaPrime, α > zero(α))
```

Comment:
I think `α > 0` is always save to write?

Reply:
Not always but here probably yes. I just copied it from above though, the check in line 36 already uses `zero` (I got the impression that `x > zero(x)` is a bit more common than `x > 0` in Distributions currently).

Reply:
Not completely sure if `x > 0` is problematic for dual or tracked numbers, it is at least for unitful numbers (which we don't officially support here though).

Reply:
Ah, I see, it fails for unitfuls. It's not crucial, more a curiosity.